.github/workflows/build.yml

```diff
@@ -63,189 +70,198 @@ jobs:
             xkb-data \
             xorg-dev
 
-          pip3 install wheel setuptools
-          pip3 install conan mako
+          pip3 install --break-system-packages wheel setuptools
+
+      - name: Install Vulkan SDK
+        shell: bash
+        run: |
+          # Download Vulkan SDK
+          curl -LS -o vulkansdk.tar.gz \
+            https://sdk.lunarg.com/sdk/download/${{ env.INEXOR_VULKAN_VERSION }}/linux/vulkansdk-linux-x86_64-${{ env.INEXOR_VULKAN_VERSION }}.tar.gz
+          # Create Vulkan SDK directory and extract
+          mkdir "${{ env.INEXOR_VULKAN_SDK_PATH }}"
+          tar xfz vulkansdk.tar.gz -C "${{ env.INEXOR_VULKAN_SDK_PATH }}"
 
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: Cache Conan Dependencies
-        uses: actions/cache@v3
-        with:
-          path: ${{ env.CONAN_USER_HOME }}
-          key: conan-${{ runner.os }}-${{ matrix.config.name }}-${{ matrix.config.build_type }}-${{ hashFiles('conanfile.py') }}
-
-      - name: Configure CMake
+      - name: Configure CMake (Debug and Release)
         shell: bash
         run: |
           export CC=${{ matrix.config.cc }}
           export CXX=${{ matrix.config.cxx }}
-          export PATH="${{ env.inexor_conan_path }}":$PATH
-
-          # Setup conan
-          # Note: libstdc++11 is needed to use new libc++ ABI
-          conan profile new default --detect --force
-          conan profile update settings.compiler.libcxx=libstdc++11 default
-
-          # Configure cmake
+          export VULKAN_SDK="${{ env.INEXOR_VULKAN_SDK_PATH }}/${{ env.INEXOR_VULKAN_VERSION }}/x86_64"
+          export PATH=$VULKAN_SDK/bin:$PATH
+          export LD_LIBRARY_PATH=$VULKAN_SDK/lib:$LD_LIBRARY_PATH
+          export VK_LAYER_PATH=$VULKAN_SDK/etc/explicit_layer.d
```

Comment:
I am not sure if we need this. I removed it from the build and static analysis script (the only two where I found it), and let's see if they fail.

Most of these things come from just testing and trying to get the CI to work because I can't really test the CI workflows locally.

Reply:
Ok it works without it, I will squash that commit.